lib/Role/utils.js

```diff
@@ -0,0 +1,167 @@
+export function getUpdatedDisabledCapabilities(newDisabledList, initialDisabledCapabilities, isChecked) {
```

Comment:
Header comments on each of these functions would be really nice. Just a sentence is fine. Ideally, it's a sentence or two, plus descriptions of each argument and its type, and the shape of the return value.